kairon/actions/definitions/email.py

```diff
@@ -100,3 +100,14 @@ async def execute(self, dispatcher: CollectingDispatcher, tracker: Tracker, doma
             ).save()
         dispatcher.utter_message(bot_response)
         return {KaironSystemSlots.kairon_action_response.value: bot_response}
+
+    @staticmethod
+    def __get_from_email_and_to_email(action_config: EmailActionConfig, tracker: Tracker):
+        from_email_type = action_config['from_email']['parameter_type']
+        to_email_type = action_config['to_email']['parameter_type']
+        from_email = tracker.get_slot(action_config['from_email']['value']) if from_email_type == "slot" \
+            else action_config['from_email']['value']
+        to_email = tracker.get_slot(action_config['to_email']['value']) if to_email_type == "slot" \
+            else action_config['to_email']['value']
+        to_email = [to_email] if isinstance(to_email, str) else to_email
```

Comment:
add a validation that the to_mail must be of type str and list only and from_mail must be of type str only